packages/@angular/cli/blueprints/ng/files/karma.conf.js

```diff
@@ -15,25 +15,14 @@ module.exports = function (config) {
     client:{
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
-    files: [
```

Comment:
No need to regenerate projects. `tests/e2e/tests/test/test-backwards-compat.ts` tests that the old config work. 

The new plugin has some logic that removes these entries if they are present in the karma config, that ensures backwards compatibility.